applications_tests/lethe-fluid/generators/heat_transfer_vof_lpbf_benchmark_box_ref_generator.prm

```diff
@@ -226,17 +226,19 @@ subsection VOF
     set verbosity = quiet
     set beta      = 20
   end
-  subsection interface sharpening
-    set enable                  = true
-    set threshold               = 0.5
-    set interface sharpness     = 1.5
-    set frequency               = 10
-    set type                    = adaptive
-    set threshold max deviation = 0.4
-    set max iterations          = 50
-    set monitored fluid         = fluid 1
-    set tolerance               = 1e-7
-    set verbosity               = quiet
+  subsection interface regularization method
+    set type      = sharpening
+    set frequency = 10
+    set verbosity = quiet
+    subsection interface sharpening
```

Comment:
While we are at it do we want to change the name of the interface sharpening subsection? What is the official name you are using for it in the article?

Reply:
We use projection-based. I can do `projection interface reinitialization` for the subsection name? It follows the same naming convention as the algebraic and geometric one

Reply:
However, in the article of Aliabadi and Tezduyar it is named interface sharpening.

Reply:
I'll you judge on this, all these names make perfect sense to me so go with what you prefer :)